### ISSUE TITLE
Quotes are lost in webpack.config.js

### ISSUE THREAD

**Issue by peteruithoven** [ryanmurakami/webpack-addons-react#3]
When running this with webpack-cli `1.3.3` the generated webpack.config.js has issues, like it generates webpack.config.js without some of the quores:
``` js
module.exports = {
  entry: './src/index.js',

  output: {
    filename: '[name].js'
  },

  context: path.join(__dirname, "src"),

  resolve: {
    extensions: [.js, .jsx]
  },

  module: {
    rules: [{
      test: /\.jsx?$/,
      exclude: /node_modules/,
      loader: babel-loader,

      options: {
        presets: [es2015, react]
      }
    }]
  }
}
```

```
$ webpack
/home/peteruithoven/Projects/www/webpack/webpack3/webpack-cli-init-react/webpack.config.js:11
    extensions: [.js, .jsx]
                 ^
SyntaxError: Unexpected token .
    at createScript (vm.js:56:10)
    at Object.runInThisContext (vm.js:97:10)
    at Module._compile (module.js:542:28)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at requireConfig (/usr/lib/node_modules/webpack/bin/convert-argv.js:97:18)
```

**Comment by peteruithoven** [comment]
Seeing that the output.filename is generated correctly I think the `"`'s need to be added around `'`'s in `dev-config.js`?

**Comment by a-x-** [comment]
ping

**Comment by ryanmurakami** [comment]
Thanks for opening the issue @peteruithoven! 
@a-x- resolved this with #7 and I published the fix with `0.1.5`. Let me know if you have any other issues!
